.ci.yaml

```diff
@@ -3335,6 +3335,33 @@ targets:
       - bin/**
       - .ci.yaml
 
+  - name: Mac framework_tests_impeller
+    recipe: flutter/flutter_drone
+    timeout: 60
+    bringup: true
+    properties:
+      cpu: x86 # https://github.com/flutter/flutter/issues/119880
```

Comment:
should the name just be `Mac_arm64 framework_tests_impeller` instead? cc @godofredoc